guidelines/technical-guidelines/08-telepresence-guide.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Using Telepresence for local Kyma development
+---
+
+This document is a general guide to local development with [Telepresence](https://www.telepresence.io/) (tested on version `0.101`).
+
+The Kyma component that you want to debug stores its state in the Kubernetes Custom Resource and, therefore, depends on Kubernetes.    
+Mocking the dependency and developing locally are not possible, and manual deployment on every change is a mundane task.  
```

Comment:
```suggestion
Mocking the dependency and developing locally is not possible, and manual deployment on every change is a mundane task.  
```

Reply:
I asked about it before and we treat these two things independently, so `are`.